README.md

```diff
@@ -216,6 +216,53 @@ files for all the artwork we've created is available, although we
 haven't yet put it online -- yell if you need it as a basis for making
 things like orbital diagrams.)
 
+### Images
+
+The mission images folder (eg `missions/a11/images`) contains a number
+of images in standard locations and of standard sizes. This is an
+incomplete list. There are Photoshop templates available to help with
+constructing some of these (talk to the mailing list to get hold of
+them).
+
+ * `badge.png` (200 pixels wide, square-ish) and `badge_thumb.png` (40x40
+   pixels): the mission patch, with a transparent background, designed
+   for use on a dark background
+ * `avatars/` contains images for each character in the transcript, in
+   one of the following forms:
+     * transcript name: `F.png`, `IWO_48.png`
+     * ground crew or similar: `capcom_generic.jpg`, `charlie_duke.png`, `nixon.png`, in black and white
+     * astronauts in flight: `aldrin.jpg`, `armstrong.jpg`, in black and white with a yellow filter applied
+     * `blank_avatar_48.png` (default blank avatar)
```

Comment:
Maybe link to https://github.com/spacelog/spacelog/wiki/avatars somewhere in this section?